src/ExpenseTrackerGrupo2/DataAccess/Repositories/Concretes/BaseRepository.cs

```diff
@@ -0,0 +1,72 @@
+using System.Reflection;
+using Dapper;
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.Data;
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.Utils;
+
+namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Concretes;
+public class BaseRepository<T> : IBaseRepository<T> where T : class
+{
+    protected readonly IDbConnectionFactory _dbConnectionFactory;
+
+    public BaseRepository(IDbConnectionFactory dbConnectionFactory)
+    {
+        _dbConnectionFactory = dbConnectionFactory;
+    }
+
+    private string GetColumnNames()
```

Comment:
@gabriela-garcia-jalafund 
Did you forget to move theses functions? Or will they stay here?

Reply:
Those methods do need to be there.